### PR TITLE
GSoC proposal for the GeoStats.jl framework

### PR DESCRIPTION
**Pull request by juliohm**
Sorry for the late notice. I've just seen the announcement on Discourse. Hope there is time still 🙏🏽

### CODE REVIEW

**Comment by logankilpatrick** [comment]
Nice! Can you add a difficult for each project?

**Comment by juliohm** [comment]
Will update the PR tomorrow morning (midnight here in Brazil). Will add the difficulty level and then the link to the main JSOC page.

File: jsoc/gsoc/GeoStats.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- [Unsupervised classification of multivariate geostatistical data: Two algorithms](https://www.sciencedirect.com/science/article/pii/S0098300415001314)
+- [A density-based spatial clustering algorithm considering both spatial proximity and attribute similarity](https://www.sciencedirect.com/science/article/pii/S0098300411004419)
+@@
+
+## New geostatistical simulation methods
+
+Geostatistical simulation consists of generating multiple alternative realizations of geospatial data according to a given geospatial distribution.
+The litetaure on simulation methods is vast, but a few of them are particularly useful.
+
+The goal of this project is to implement geostatistical simulation method from the geostatistics literature using the GeoStats.jl API.
```
Comment (juliohm) [inline jsoc/gsoc/GeoStats.md:29]:
```suggestion
The goal of this project is to implement a geostatistical simulation method from the geostatistics literature using the GeoStats.jl API.
```

File: jsoc/gsoc/GeoStats.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## New geostatistical clustering methods
+
+Statistical clustering cannot be applied straightforwardly to geospatial data. Geospatial constraints require clusters to be
+contiguous volumes in the map, something that is not taken into account by traditional methods (e.g. K-Means, Spectral Clustering).
+
+The goal of this project is to implement a geospatial clustering method from the geostatistics literature using the GeoStats.jl API.
+
+**Desired skills:** Statistics, Clustering, Graph Theory
+
+**References:**
```
Comment (juliohm) [inline jsoc/gsoc/GeoStats.md:17]:
```suggestion
**Desired skills:** Statistics, Clustering, Graph Theory

**Difficulty level:** Medium

**References:**
```

File: jsoc/gsoc/GeoStats.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## New geostatistical simulation methods
+
+Geostatistical simulation consists of generating multiple alternative realizations of geospatial data according to a given geospatial distribution.
+The litetaure on simulation methods is vast, but a few of them are particularly useful.
+
+The goal of this project is to implement a geostatistical simulation method from the geostatistics literature using the GeoStats.jl API.
+
+**Desired skills:** Geostatistics, Stochastics, HPC
+
+**References:**
```
Comment (juliohm) [inline jsoc/gsoc/GeoStats.md:35]:
```suggestion
**Desired skills:** Geostatistics, Stochastics, HPC

**Difficulty level:** Hard

**References:**
```

File: jsoc/gsoc/GeoStats.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+## Migrate from Plots.jl to Makie.jl recipes
+
+The project currently relies on Plots.jl recipes to visualize geospatial data sets as well as many other objects defined in the framework.
+However, very large data sets (e.g. 3D volumes) cannot be visualized easily. The Makie.jl project is a promissing alternative.
+
+The goal of this project is to migrate all plot recipes from Plots.jl to Makie.jl.
+
+**Desired skills:** Visualization, Plotting, Geometry, HPC, GPU
+
+## How to get started?
```
Comment (juliohm) [inline jsoc/gsoc/GeoStats.md:52]:
```suggestion
**Desired skills:** Visualization, Plotting, Geometry, HPC, GPU

**Difficulty level:** Medium

## How to get started?
```

**Comment by juliohm** [comment]
I think I managed to fix the issue quickly on GitHub directly. Please let me know if something else needs to be improved.

**Comment by logankilpatrick** [comment]
Looks great! I suggest trying to find a co-mentor for these projects as well

**Comment by juliohm** [comment]
Awesome! I'll contact a maintainer who may be interested. Thanks for the
quick response.

**Pull request merged**